packages/api-server/src/price-oracle.ts

```diff
@@ -27,6 +27,10 @@ export const CKB_PRICE_CACHE_KEY = "priceOracle:ckbUsd";
 // gas price cache
 const GAS_PRICE_CACHE_KEY = `priceOracle:gasPrice`;
 
+const BLOCK_CONGESTION_GAS_USED = BigInt(
+  envConfig.blockCongestionGasUsed || "33848315"
```

Comment:
Please add a comment explaining why we use this default value.

Reply:
updated